### PR TITLE
Check 811 failures in CI(do not merge)

### PR DESCRIPTION
**Pull request by czyzm**
DO NOT MERGE

---
This change is [Reviewable](https://reviewable.io/reviews/thaliproject/thali_cordovaplugin/815)

### CODE REVIEW

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (26c90ec)

**Comment by ThaliTester** [comment]
Test (Fail) 79751015 build is completed (26c90ec)

See https://github.com/ThaliTester/TestResults/tree/7975101526c90ec_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (7282e1a)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (7282e1a)

See https://github.com/ThaliTester/TestResults/tree/797510157282e1a_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 797510157282e1a(7282e1a) has failed

See https://github.com/ThaliTester/TestResults/tree/797510157282e1a_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 4. task. (0d7f48d)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (0d7f48d)

See https://github.com/ThaliTester/TestResults/tree/797510150d7f48d_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 797510150d7f48d(0d7f48d) has failed

See https://github.com/ThaliTester/TestResults/tree/797510150d7f48d_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 2. task. (0318bb5)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (0318bb5)

See https://github.com/ThaliTester/TestResults/tree/797510150318bb5_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 797510150318bb5(0318bb5) has failed

See https://github.com/ThaliTester/TestResults/tree/797510150318bb5_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by andrew-aladev** [comment]
Let me say what I think about this pr:

Let me say what I think about this pr:

1. `thali/install/setUpTests.sh` and `thali/install/setUpDesktop.sh` have obtained a custom pouch db and android unittests dir.
2. `thali/NextGeneration/thaliWifiInfrastructure.js` has jsdoc fixed and `BOGUS_*` deleted. I don't know the reason why this exists in master. Our working branch vNext_yarong_417_1 doesn't have this thing.
3. We don't care about `thali/NextGeneration/thaliReplicationManager.js` because it will be replaced with new one.
4. `thali/NextGeneration/thaliPeerPool/thaliPeerPoolDefault.js` has obtained an ability to stop its instance. It looks fine.
5. `thali/NextGeneration/thaliMobileNativeWrapper.js` has `BOGUS_*` deleted.
6. `thali/NextGeneration/thaliConfig.js` had `BOGUS_*` deleted and some options added. We are already using these options in vNext_yarong_417_1.
7. We don't care about `thali/NextGeneration/security/thaliACLLayer.js` because it will be deleted.
8. `thali/NextGeneration/replication/thaliSendNotificationBasedOnReplication.js` obtained functions `getPskIdToSecret` and `getPskIdToPublicKey` that we are already using.
9. `thali/NextGeneration/replication/thaliReplicationPeerAction.js` that's the file I am working with every day. This file gave me a 'race condition' that we were talking about several days ago. We can implement more presize errorArray handling and verbose printing in future. We should definitely write many tests for this code in future. But the code itself looks fine.
10. `thali/NextGeneration/replication/thaliPullReplicationFromNotification.js` has obtained an ability to stop its client. It creates ReplicationPeerAction and it is going to enqueue it to PeerPoolInterface. It uses hash to kill existing ReplicationPeerAction with the same connection type and id.
11. `thali/NextGeneration/replication/localSeqManager.js` is going to sync last replication version between peers. I don't like this code, but I don't know how to make refactoring. It is not readable: timers are jumping around promises with strange self variables and overcomplicated scopes. I can investigate this in future.
12. `thali/NextGeneration/notification/thaliNotificationServer.js` has fixed getPskIdToSecret, that we are already using.
13. `thali/NextGeneration/notification/thaliNotificationClient.js` obtained a way to ignore empty beacon data.
25. `thali/NextGeneration/notification/thaliNotificationAction.js` has added superclass kill on complete.
26. `thali/NextGeneration/makeIntoCloseAllServer.js` has fixed connections array. Splice removes closed connection.
27. `test/www/jxcore/lib/testUtils.js` has added test server with participants that we are using today. The code is ok.
28. `test/www/jxcore/lib/httpTester.js` is a helper for testing `thali/NextGeneration/replication/localSeqManager.js`
29. `test/www/jxcore/bv_tests/testthaliPeerPoolDefault.js` has `BOGUS_*` deleted and tested stop of PeerPoolDefault.
30. `test/www/jxcore/bv_tests/testThaliReplicationPeerActionCoordinated.js` and `test/www/jxcore/bv_tests/testThaliReplicationPeerAction.js` tested a ReplicationPeerAction. The code seems fine.
31. `test/www/jxcore/bv_tests/testThaliPullReplicationFromNotificationCoordinated.js` and `test/www/jxcore/bv_tests/testThaliPullReplicationFromNotification.js` tested a ReplicationPeerAction. The code seems fine.

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (f6cefdf)

**Comment by ThaliTester** [comment]
Test (Fail) 79751015 build is completed (f6cefdf)

See https://github.com/ThaliTester/TestResults/tree/79751015f6cefdf_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

File: thali/NextGeneration/notification/thaliNotificationClient.js [hunk 1]
```diff
@@ -98,6 +101,17 @@ function ThaliNotificationClient(thaliPeerPoolInterface, ecdhForLocalDevice) {
 util.inherits(ThaliNotificationClient, EventEmitter);
 
 /**
+ * Fired whenever we discover a peer who is looking for us.
+ *
+ * @public
+ * @event module:thaliNotificationClient.event:peerAdvertisesDataForUs
+ * @type {PeerAdvertisesDataForUs}
+ */
+ThaliNotificationClient.prototype.Events = {
```
Comment (andrew-aladev) [inline thali/NextGeneration/notification/thaliNotificationClient.js]: Can we use static object here?

File: thali/NextGeneration/replication/localSeqManager.js [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+LocalSeqManager.prototype._doImmediateSeqUpdate = function (seq) {
+  var self = this;
+
+  if (self._stopCalled) {
+    return Promise.reject(new Error('Stop Called'));
+  }
+
+  self._lastUpdateTime = Date.now();
+
+  function getRevOrNull() {
```
Comment (andrew-aladev) [inline thali/NextGeneration/replication/localSeqManager.js]: We can create a private method without arguments _getRevision here.

File: thali/NextGeneration/replication/localSeqManager.js [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+      })
+      .catch(function (err) {
+        if (err.status === 404) {
+          return null;
+        }
+        return Promise.reject(err);
+      });
+  }
+
+  return (self._seqDocRev ? Promise.resolve(self._seqDocRev) : getRevOrNull())
```
Comment (andrew-aladev) [inline thali/NextGeneration/replication/localSeqManager.js]: We can move this line into _getRevision method.

File: thali/NextGeneration/replication/thaliPullReplicationFromNotification.js [hunk 4]
```diff
@@ -9,37 +15,125 @@
...
+  this._started = false;
 }
 
+ThaliPullReplicationFromNotification.prototype._started = null;
+
+ThaliPullReplicationFromNotification.prototype._thaliNotificationClient = null;
+
+ThaliPullReplicationFromNotification.prototype._peerDictionary = null;
+
+ThaliPullReplicationFromNotification.prototype._peerDictionaryKey =
```
Comment (andrew-aladev) [inline thali/NextGeneration/replication/thaliPullReplicationFromNotification.js]: This function asks to be static.

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (a95812e)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (a95812e)

See https://github.com/ThaliTester/TestResults/tree/79751015a95812e_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 79751015a95812e(a95812e) has failed

See https://github.com/ThaliTester/TestResults/tree/79751015a95812e_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (2cd5224)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (2cd5224)

See https://github.com/ThaliTester/TestResults/tree/797510152cd5224_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 797510152cd5224(2cd5224) has failed

See https://github.com/ThaliTester/TestResults/tree/797510152cd5224_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (d1afc27)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (d1afc27)

See https://github.com/ThaliTester/TestResults/tree/79751015d1afc27_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 3. task. (abe04ee)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (abe04ee)

See https://github.com/ThaliTester/TestResults/tree/79751015abe04ee_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 79751015d1afc27(d1afc27) has failed

See https://github.com/ThaliTester/TestResults/tree/79751015d1afc27_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
Test 79751015abe04ee(abe04ee) has failed

See https://github.com/ThaliTester/TestResults/tree/79751015abe04ee_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (d1e118e)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (d1e118e)

See https://github.com/ThaliTester/TestResults/tree/79751015d1e118e_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 79751015d1e118e(d1e118e) has failed

See https://github.com/ThaliTester/TestResults/tree/79751015d1e118e_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (e87fad4)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (e87fad4)

See https://github.com/ThaliTester/TestResults/tree/79751015e87fad4_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 79751015e87fad4(e87fad4) has failed

See https://github.com/ThaliTester/TestResults/tree/79751015e87fad4_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 2. task. (8320e75)

**Comment by ThaliTester** [comment]
Test (Fail) 79751015 build is completed (8320e75)

See https://github.com/ThaliTester/TestResults/tree/797510158320e75_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (3577c7c)

**Comment by ThaliTester** [comment]
Test (Fail) 79751015 build is completed (3577c7c)

See https://github.com/ThaliTester/TestResults/tree/797510153577c7c_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (3cc0747)

**Comment by ThaliTester** [comment]
Test (Fail) 79751015 build is completed (3cc0747)

See https://github.com/ThaliTester/TestResults/tree/797510153cc0747_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (a09e88e)

**Comment by ThaliTester** [comment]
Test (Fail) 79751015 build is completed (a09e88e)

See https://github.com/ThaliTester/TestResults/tree/79751015a09e88e_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 2. task. (cf21110)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (cf21110)

See https://github.com/ThaliTester/TestResults/tree/79751015cf21110_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 79751015cf21110(cf21110) has failed

See https://github.com/ThaliTester/TestResults/tree/79751015cf21110_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (931ffdc)

**Comment by ThaliTester** [comment]
Test (Fail) 79751015 build is completed (931ffdc)

See https://github.com/ThaliTester/TestResults/tree/79751015931ffdc_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (afaf1d7)

**Comment by ThaliTester** [comment]
Test (Fail) 79751015 build is completed (afaf1d7)

See https://github.com/ThaliTester/TestResults/tree/79751015afaf1d7_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (5a6c85b)

**Comment by ThaliTester** [comment]
Test (Fail) 79751015 build is completed (5a6c85b)

See https://github.com/ThaliTester/TestResults/tree/797510155a6c85b_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (c4115df)

**Comment by ThaliTester** [comment]
Test (Fail) 79751015 build is completed (c4115df)

See https://github.com/ThaliTester/TestResults/tree/79751015c4115df_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (6040d40)

**Comment by ThaliTester** [comment]
Test (Fail) 79751015 build is completed (6040d40)

See https://github.com/ThaliTester/TestResults/tree/797510156040d40_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (77d639c)

**Comment by ThaliTester** [comment]
Test (Fail) 79751015 build is completed (77d639c)

See https://github.com/ThaliTester/TestResults/tree/7975101577d639c_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 2. task. (ccfe79c)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (ccfe79c)

See https://github.com/ThaliTester/TestResults/tree/79751015ccfe79c_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 79751015ccfe79c(ccfe79c) has failed

See https://github.com/ThaliTester/TestResults/tree/79751015ccfe79c_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (7a5d8bb)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (7a5d8bb)

See https://github.com/ThaliTester/TestResults/tree/797510157a5d8bb_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 797510157a5d8bb(7a5d8bb) has successfully finished without an error

See https://github.com/ThaliTester/TestResults/tree/797510157a5d8bb_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (9390029)

See https://github.com/ThaliTester/TestResults/tree/797510159390029_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 797510159390029(9390029) has failed

See https://github.com/ThaliTester/TestResults/tree/797510159390029_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (c29d5b7)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (c29d5b7)

See https://github.com/ThaliTester/TestResults/tree/79751015c29d5b7_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 79751015c29d5b7(c29d5b7) has failed

See https://github.com/ThaliTester/TestResults/tree/79751015c29d5b7_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (4a934e8)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (4a934e8)

See https://github.com/ThaliTester/TestResults/tree/797510154a934e8_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 797510154a934e8(4a934e8) has failed

See https://github.com/ThaliTester/TestResults/tree/797510154a934e8_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 3. task. (05baaca)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (05baaca)

See https://github.com/ThaliTester/TestResults/tree/7975101505baaca_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 7975101505baaca(05baaca) has failed

See https://github.com/ThaliTester/TestResults/tree/7975101505baaca_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 2. task. (81c1e09)

**Comment by ThaliTester** [comment]
Test (Fail) 79751015 build is completed (81c1e09)

See https://github.com/ThaliTester/TestResults/tree/7975101581c1e09_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 3. task. (c075caf)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (c075caf)

See https://github.com/ThaliTester/TestResults/tree/79751015c075caf_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 79751015c075caf(c075caf) has failed

See https://github.com/ThaliTester/TestResults/tree/79751015c075caf_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (d87d04c)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (d87d04c)

See https://github.com/ThaliTester/TestResults/tree/79751015d87d04c_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 79751015d87d04c(d87d04c) has failed

See https://github.com/ThaliTester/TestResults/tree/79751015d87d04c_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (a5ad772)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (a5ad772)

See https://github.com/ThaliTester/TestResults/tree/79751015a5ad772_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 79751015a5ad772(a5ad772) has failed

See https://github.com/ThaliTester/TestResults/tree/79751015a5ad772_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (2beef86)

**Comment by ThaliTester** [comment]
Test 797510152beef86(2beef86) has failed

See https://github.com/ThaliTester/TestResults/tree/797510152beef86_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (6960f45)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (6960f45)

See https://github.com/ThaliTester/TestResults/tree/797510156960f45_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 797510156960f45(6960f45) has failed

See https://github.com/ThaliTester/TestResults/tree/797510156960f45_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (7fdac7f)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (7fdac7f)

See https://github.com/ThaliTester/TestResults/tree/797510157fdac7f_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 797510157fdac7f(7fdac7f) has failed

See https://github.com/ThaliTester/TestResults/tree/797510157fdac7f_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (a3268dd)

**Comment by ThaliTester** [comment]
Test (Fail) 79751015 build is completed (a3268dd)

See https://github.com/ThaliTester/TestResults/tree/79751015a3268dd_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (ab10927)

**Comment by ThaliTester** [comment]
Test (Fail) 79751015 build is completed (ab10927)

See https://github.com/ThaliTester/TestResults/tree/79751015ab10927_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 2. task. (13b55b4)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (13b55b4)

See https://github.com/ThaliTester/TestResults/tree/7975101513b55b4_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 7975101513b55b4(13b55b4) has failed

See https://github.com/ThaliTester/TestResults/tree/7975101513b55b4_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 2. task. (67d962b)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (67d962b)

See https://github.com/ThaliTester/TestResults/tree/7975101567d962b_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 7975101567d962b(67d962b) has failed

See https://github.com/ThaliTester/TestResults/tree/7975101567d962b_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (49b9187)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (49b9187)

See https://github.com/ThaliTester/TestResults/tree/7975101549b9187_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 7975101549b9187(49b9187) has failed

See https://github.com/ThaliTester/TestResults/tree/7975101549b9187_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (1e03ec5)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (1e03ec5)

See https://github.com/ThaliTester/TestResults/tree/797510151e03ec5_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 797510151e03ec5(1e03ec5) has failed

See https://github.com/ThaliTester/TestResults/tree/797510151e03ec5_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (f24a8ad)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (f24a8ad)

See https://github.com/ThaliTester/TestResults/tree/79751015f24a8ad_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 79751015f24a8ad(f24a8ad) has failed

See https://github.com/ThaliTester/TestResults/tree/79751015f24a8ad_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 2. task. (1684451)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (1684451)

See https://github.com/ThaliTester/TestResults/tree/797510151684451_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 797510151684451(1684451) has failed

See https://github.com/ThaliTester/TestResults/tree/797510151684451_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (9e44f0b)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (9e44f0b)

See https://github.com/ThaliTester/TestResults/tree/797510159e44f0b_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 797510159e44f0b(9e44f0b) has failed

See https://github.com/ThaliTester/TestResults/tree/797510159e44f0b_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Comment by ThaliTester** [comment]
PR is added to the queue for testing as 1. task. (007586f)

**Comment by ThaliTester** [comment]
Test (Success) 79751015 build is completed (007586f)

See https://github.com/ThaliTester/TestResults/tree/79751015007586f_Check_811_failures_in_CI_do_not_merge__czyzm/ for the logs

**Comment by ThaliTester** [comment]
Test 79751015007586f(007586f) has failed

See https://github.com/ThaliTester/TestResults/tree/79751015007586f_Check_811_failures_in_CI_do_not_merge__czyzm/ for the fail logs

**Pull request closed without merging**